dev/devicelab/manifest.yaml

```diff
@@ -250,6 +250,13 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  web_size__compile_test:
+    description: >
+      Measures the size of a dart2js bundle.
+    stage: devicelab
+    required_agent_capabilities: ["mac/android"]
```

Comment:
Why not linux? Linux machines are easier to manage and scale, no?

Reply:
Ahh good point, I left it as mac since that is what I tested it on